docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -0,0 +1,122 @@
+We also host Design Kits and pre-built Templates.
+
+Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.
+
+We're also making ambitious moves to incorporate our full suite of components into Toolpad, a low-code admin builder tool for assembling full-stack apps faster than ever.
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).
+
+## The culture
+
+MUI is a fully remote company with a team that spans the globe.
+The majority of our work is asynchronous, and we rely on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
+
+For additional details about the culture you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Both our open-source community and our premium products are growing fast (x2 YoY).
+We need talented people to keep that going!
+
+We are aiming to make high-end design accessible to the many, empowering developers that lack access to a designer as well as saving time for designers with a source they can trust.
+[According to our last survey](https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library), the quality of the design execution (UX & UI) is the most important criteria used by developers & designers to pick their next UI library.
+
+### Why this is interesting
+
+The advanced components portfolio is still small, with a million interesting and challenging problems to solve.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+But while we are [the leading](https://tsh.io/state-of-frontend/#over-the-past-year-which-of-the-following-design-systems-was-your-favorite-go-to-solution) UI design system in the frontend space, the adoption of MUI is only at 25%. More importantly, the challenges of developers and designers to solve when creating UIs go way beyond the ones of design systems.
+We envision a future where MUI X becomes the default toolkit for web developers to create advanced UIs.
+It's why we started Base UI, Joy UI, and MUI Toolpad. Design is key to achieving this goal.
+
+## The role
+
+### What you'll do on a day-to-day basis
+
+You will extend the [Design Engineer](https://mui-org.notion.site/Design-Engineer-09e4cac2e0bb4df3b173458b4b4b5666) responsibilities. Depending on the day, you'll:
```

Comment:
```suggestion
The Design Engineer role sits right at the intersection between product design and software engineering. So, depending on the initiative, your day can be entirely different! Here's what you'll be able to do:
```

I'd avoid adding an external link here (that doesn't add a lot, I think), given this page is already pretty long. Maybe something that would dovetail with the big list of items below, like this, would work?